src/index.tsx

```diff
@@ -880,7 +874,7 @@ export class Resizable extends React.PureComponent<ResizableProps, State> {
   }
 
   updateSize(size: Size) {
-    this.setState({ width: size.width, height: size.height });
+    this.setState({ width: size.width || 'auto', height: size.height || 'auto' });
```

Comment:
Could you please use `??` instead of `||`
```suggestion
    this.setState({ width: size.width ?? 'auto', height: size.height ?? 'auto' });
```